parsuricata/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.2.3"
+__version__ = "0.2.4"
```

Comment:
![tumblr_inline_mssg0qW1X91qz4rgp540](https://user-images.githubusercontent.com/33840/120588589-7020a400-c405-11eb-99b4-9ba0af996bb1.gif)
```suggestion
__version__ = "0.2.3"
```

Reply:
Sounds good to me